tpl/mess_hinderedrotor.tpl

```diff
@@ -1,6 +1,4 @@
       Rotor     Hindered
-      Geometry[angstrom]          {natom}
-{geom}
```

Comment:
This line caused errors in MESS, you can only have one instance of a `Geometry` section per block.